src/component-library/TokenInput/TokenInput.tsx

```diff
@@ -56,11 +66,13 @@ const TokenField = forwardRef<HTMLInputElement, TokenFieldProps>(
 
     const endAdornment = (
       <TokenAdornment>
-        <TokenFieldSymbol>{tokenSymbol}</TokenFieldSymbol>
-        <TokenFieldUSD>{`≈ ${valueInUSD}`}</TokenFieldUSD>
+        <TokenInputSymbol>{tokenSymbol}</TokenInputSymbol>
+        <TokenInputUSD>{`≈ ${valueInUSD}`}</TokenInputUSD>
       </TokenAdornment>
     );
 
+    const isOverflowing = (value?.toString().length || 0) > 9;
```

Comment:
Not sure about this way of detecting an overflowing form field - the number of characters at at which we want to reduce the text size depends on the width of the input, which is not fixed.